rust/crd/src/lib.rs

```diff
@@ -101,7 +101,8 @@ impl TrinoRole {
         let parsed_version = Version::parse(version.to_string().as_ref()).unwrap();
 
         vec![
-            format!("trino-server-{}/bin/launcher", parsed_version.patch),
+            "python3".to_string(),
```

Comment:
This means a customer with a bare-metal cluster has to have `python3` set up in their path. This requirement should also be documented somewhere in the operator docs.

Reply:
Agreed, but it doesn't look like the operator has any yet.